src/ClusterBootstrap/services/monitor/alerting/jobs.rules

```diff
@@ -0,0 +1,10 @@
+groups:
+    - name: idle-job
+      rules:
+      - alert: low-gpu-usage
+        expr: avg(task_gpu_percent) by (pod_name, instance, user_email) == 0
+        for: 4h
+        labels:
+          type: user_alert
+        annotations:
+          summary: "Your job {{$labels.pod_name}} in {{$labels.instance}} have not used gpu for 4 hours. Please kill the job if you do not need it anymore"
```

Comment:
1. Is it possible to add a link to the Job?
2. For distributed jobs, will each idle pod trigger an email? If so, is it possible to aggregate them at job level?

Reply:
I see that it's already assembled into one email. But still can we make them cleaner by grouping them into jobs?

Reply:
1. Yes, we can add a link, but to do so, alert manager must know webportal's url, maybe hub's, how can we get it? Through config?
2. Yes, in that case, we avg all task role's gpu usage and still check if it is 0?

Reply:
Also for 1, where the link pointing to? Can we assume all jobs are from hub instead of cluster local webportal?